src/provider/CentralServerProvider.tsx

```diff
@@ -117,24 +117,29 @@ export default class CentralServerProvider {
 
   // eslint-disable-next-line class-methods-use-this
   public getTenants(): Partial<Tenant>[] {
-    if (__DEV__) {
-      return [
-        { subdomain: 'testcharger', name: 'SAP Labs New Charging Stations Tests' },
-        { subdomain: 'testperf', name: 'SAP Labs Performance Tests' },
-        { subdomain: 'demopricing', name: 'SAP Labs Demo for Pricing' },
-        { subdomain: 'demobilling', name: 'SAP Labs Demo for Billing' },
-        { subdomain: 'slf', name: 'SAP Labs France (prod)' },
-        { subdomain: 'slfcah', name: 'SAP Labs France (Charge@Home) (prod)' },
+    let tenants = SecuredStorage.getTenants();
+    if (tenants) {
```

Comment:
I fail to understand that test: you retrieve a tenants array just before and overwrite it just after with default array if the array is not empty, it's meaningless. You should build a default tenants array only if no tenants array have been saved locally on the device.